cg/constants/demultiplexing.py

```diff
@@ -18,6 +19,15 @@ class FlowCellType(StrEnum):
     HISEQ: str = "hiseq"
 
 
+class SequencingInstruments(StrEnum):
+    """Define properties of the sequencing instruments."""
+
+    NOVASEQXPLUS: str = "NovaSeqXPlus"
+    NOVASEQ6000: str = "NovaSeq6000"
+    NOVA_SEQ_6000_NAMES: List[str] = ["A00621", "A00187", "A00689", "A01901"]
```

Comment:
Novaseq is spelled in many ways here. Let's converge on one. Is there not already a constant with the names?